tests/phpunit/api/v3/RelationshipTest.php

```diff
@@ -779,6 +779,61 @@ public function testRelationshipsGet() {
   }
 
   /**
+   * Chain Relationship.get and to Contact.get.
```

Comment:
This test passed before applying the fix. I put it inhere while trying to insulate the problem.